src/Todoist.Net/Models/CollaboratorState.cs

```diff
@@ -0,0 +1,43 @@
+using Newtonsoft.Json;
+
+namespace Todoist.Net.Models
+{
+    /// <summary>
+    /// Represents a Todoist collaborator state.
+    /// </summary>
+    public class CollaboratorState
+    {
+        internal CollaboratorState()
+        {
+        }
+
+        /// <summary>
+        /// Gets the user id of the collaborator.
+        /// </summary>
+        /// <value>The user id.</value>
+        [JsonProperty("user_id")]
+        public string UserId { get; internal set; }
+
+        /// <summary>
+        /// Gets the shared project id.
+        /// </summary>
+        /// <value>The project id.</value>
+        [JsonProperty("project_id")]
+        public string ProjectId { get; internal set; }
+
+        /// <summary>
+        /// Gets the collaborator state.
+        /// </summary>
+        /// <value>The state.</value>
+        [JsonProperty("state")]
+        public string State { get; internal set; }
```

Comment:
Let's create a string enum for this

Reply:
Done 👍
I created a new StringEnum class called `CollaboratorStatus`